apps/src/StudioApp.js

```diff
@@ -2279,7 +2279,8 @@ StudioApp.prototype.handleEditCode_ = function(config) {
   }
 
   // Remove maker API blocks from palette, unless maker APIs are enabled.
-  if (!project.getMakerAPIs()) {
+  // Also check config.level to ensure maker toolkit shows when editing start mode
```

Comment:
i think this comment is misleading -- `config.level.makerlabEnabled` and start mode should be decoupled, so maybe this should be something like "any level with maker enabled"